payment_allpay/__init__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please use simple headers for all py files following OCA templates:
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3